Podfile

```diff
@@ -13,9 +13,10 @@ use_frameworks!
 # - `{ :specHash => {sdk spec hash}` to depend on specific pod options (:git => …, :podspec => …) for MatrixSDK repo. Used by Fastfile during CI
 #
 # Warning: our internal tooling depends on the name of this variable name, so be sure not to change it
-$matrixSDKVersion = '= 0.20.16'
+# $matrixSDKVersion = '= 0.20.16'
 # $matrixSDKVersion = :local
 # $matrixSDKVersion = { :branch => 'develop'}
+$matrixSDKVersion = { :branch => 'langleyd/5292_refresh_tokens'}
```

Comment:
Seems forgotten

Reply:
I had it in so that this PR would build against my SDK PR so that I could build the app on CI and have diawi alpha build to distribute to people to test. Will be removing on merge.